rero_ils/theme/templates/security/email/reset_notice.txt

```diff
@@ -0,0 +1,9 @@
+{%- if user.profile.first_name and user.profile.last_name %}
+<p>{{ _('Dear') }} {{ user.profile.first_name }} {{ user.profile.last_name }},</p>
+{%- else %}
+<p>{{ _('Dear patron') }},</p>
+{%- endif %}
+<p>{{ _('Your password has been successfully reset.') }}</p>
```

Comment:
TODO: in a futur, we need a message to explain what to do if the patron did not change his password. (Hacking)